indico/modules/rb_new/operations/rooms.py

```diff
@@ -90,6 +95,40 @@ def get_managed_room_ids(user):
         return {r.id for r in Room.get_owned_by(user)}
 
 
+@memoize_redis(3600)
+def get_room_statistics(room):
+    data = {
+        'count': {
+            'id': 'times_booked',
+            'values': [],
+            'note': None
+        },
+        'percentage': {
+            'id': 'occupancy',
+            'values': [],
+            'note': _('excluding weekends')
```

Comment:
If the result is memoized on Redis, the translated string will be as well. Either you add it later in the RH or directly on the client.

Reply:
huh weird, i could have sworn i already commented somewhere that this should be handled on the client..